src/libs/runtime-utils.js

```diff
@@ -164,10 +169,24 @@ export const machineCost = machineType => {
 
 export const currentApp = _.flow(trimAppsOldestFirst, _.last)
 
-export const currentDataDisk = (app, galaxyDataDisks) => {
+export const currentAttachedDataDisk = (app, galaxyDataDisks) => {
   return _.find({ name: app?.diskName }, galaxyDataDisks)
 }
 
+export const getCurrentPersistentDiskName = (apps, galaxyDataDisks) => {
+  const currentGalaxyApp = currentApp(apps)
+  const currentDataDiskName = currentGalaxyApp?.diskName
+  const attachedDataDiskNames = _.without([undefined], _.map(app => app.diskName, apps))
+  return currentDataDiskName ?
+    _.find({ currentDataDiskName }, galaxyDataDisks) :
+    _.last(_.sortBy('auditInfo.createdDate', _.filter(({ name, status }) => status !== 'Deleting' && !_.includes(name, attachedDataDiskNames), galaxyDataDisks)))
+}
+
+export const trimDisksOldestFirst = _.flow(
+  _.remove({ status: 'Deleting' }),
+  _.sortBy('auditInfo.createdDate')
+)
+
```

Comment:
Is this method used anywhere?

Reply:
meant to delete - thought it was needed but it is not. will remove